packages/components/src/systems/PharmacySelect/PatientDetails.tsx

```diff
@@ -59,7 +74,13 @@ export function PatientDetails(props: PatientDetailsProps) {
         size="sm"
         sampleLoadingText="111 222 3333"
       >
-        {patient()?.phone}
+        <Show when={preferredPharmacy()}>
+          {preferredPharmacy()?.name}{' '}
+          <Badge size="sm" color="blue" class="ml-1">
+            Preferred
+          </Badge>
+        </Show>
+        <Show when={!preferredPharmacy()}>{patient()?.phone}</Show>
```

Comment:
If the intention is ensure these are mutually exclusive, might be worth using a `fallback` instead of two `Show`

```suggestion
        <Show when={preferredPharmacy()} fallback={patient()?.phone}>
          {preferredPharmacy()?.name}{' '}
          <Badge size="sm" color="blue" class="ml-1">
            Preferred
          </Badge>
        </Show>
```

Reply:
Yup, true, I find `fallback` a little ugly... but I you're right as that's more the correct way to do it